src/NzbDrone.Core/Organizer/FileNameValidation.cs

```diff
@@ -66,96 +65,70 @@ public static class FileNameValidation
 
     public class ValidStandardEpisodeFormatValidator : PropertyValidator
     {
-        public ValidStandardEpisodeFormatValidator()
-            : base("Must contain season and episode numbers OR Original Title")
-        {
-        }
+        protected override string GetDefaultMessageTemplate() => "Must contain season and episode numbers OR Original Title";
 
         protected override bool IsValid(PropertyValidatorContext context)
         {
-            var value = context.PropertyValue as string;
-
-            if (!FileNameBuilder.SeasonEpisodePatternRegex.IsMatch(value) &&
-                !FileNameValidation.OriginalTokenRegex.IsMatch(value))
+            if (context.PropertyValue is not string value)
```

Comment:
Here ArgumentNullException was possible and I decided to check the type.
But I am not sure about this change. In some places null is a valid value, in some cases not.
So what do u think - should I return true or false in this case?

Reply:
I think it makes sense to be false if null in this case.